src/style.css

```
@@ -21,6 +21,7 @@
 
 body {
     background-color: #ffffff;
+    color: #000091;
```

Merci beaucoup ! 🙇 

Par contre je pense que notre couleur par défaut de texte est le noir et ce bleu est réservé aux titres/liens.

```suggestion
    color: #000000;
```